locobot/robot/launch_pyro_habitat.sh

```diff
@@ -3,6 +3,13 @@
 export PYRO_SERIALIZER='pickle'
 export PYRO_SERIALIZERS_ACCEPTED='pickle'
 
+echo "Ensuring clean slate (kills python, roscore, rosmaster processes)..."
+killall -9 python &
```

Comment:
i don't think this is okay. remember that we actually can run this outside of docker (and in fact, I was going to undockerize the habitat setup in the future once we move our ROS packaging to `robostack`)

Reply:
do you mean all the killalls (including roscore, rosmaster) or just the one for python? 

In the new iteration, I've removed the `killall python` (was doing it initially to kill a languishing Pyro4 process which was giving a socket error when restarted, but turns out just exporting a `PYRO_SOCK_REUSE` variable fixes that).

Reply:
the roscore / rosmaster are probably okay for now, still kinda bad but we can live with it. The `killall python` was the really bad one .